dmt/api/auth.py

```diff
@@ -1,7 +1,12 @@
 from rest_framework import permissions
+from urlparse import urlparse
 
 
 class SafeOriginPermission(permissions.BasePermission):
     def has_permission(self, request, view):
+        safe_origin = '.columbia.edu'
```

Comment:
Not all of our domains end with .columbia.edu. Perhaps an IP-based whitelist solution might be considered here?